.ruby-version

```diff
@@ -1 +1 @@
-2.7.4
\ No newline at end of file
+3.2.2
```

Comment:
This and the WebMock update were attempts to solve the tests having a missing `uri` in the stubbed response.

That turned out to be a bug in WebMock (see `spec_helper` diff). I decided to keep them because they don't seem to hurt, but I can remove this one if you think it should be a changed for a dedicated PR.